### DISCUSSION TITLE
Error code [27] wasn't in the acceptable list

### DISCUSSION THREAD

**Discussion by atillier** [ClickHouse/clickhouse-kafka-connect#419] [General]
Hi all,

I had the connector running on a single topic fine for a while. I just added another topic, also used `topic2TableMap`, and I am getting:
`[Worker-0c75438bed646c1c3] [2024-07-26 22:00:14,448] WARN [Clickhouse|task-1] Deciding how to handle exception: Topic: [es.json], Partition: [2], MinOffset: [5370684], MaxOffset: [5370684], (QueryId: [94a2bf3d-7fe6-4759-8d74-d5e1d1d6634e]) (com.clickhouse.kafka.connect.util.Utils:69)
[Worker-0c75438bed646c1c3] [2024-07-26 22:00:14,449] WARN [Clickhouse|task-1] ClickHouseException code: 27 (com.clickhouse.kafka.connect.util.Utils:76)
[Worker-0c75438bed646c1c3] [2024-07-26 22:00:14,449] ERROR [Clickhouse|task-1] Error code [27] wasn't in the acceptable list. (com.clickhouse.kafka.connect.util.Utils:95)`

I have the table define all fine, although with a much simpler model. The JSON from the topic is like
```
{
  "timestamp": "2024-07-26T09:12:12.670907056Z",
  "event": {
          "id": 98223,
          "name": "Match #1"
   }
}
```
and the table is defined as

```
CREATE or REPLACE TABLE default."es.json" ON CLUSTER node
(
    timestamp Timestamp DEFAULT now(),
    event String
) ENGINE = ReplicatedMergeTree('/clickhouse/tables/{shard}/es.json', '{replica}')
ORDER BY (timestamp);
```

Can you help me understand what I am doing wrong?

Thanks

**Comment by mzitnik** [comment]
Can you share `topic2TableMap` configuration value

**Reply by atillier** [reply]
Sorry for the lag @mzitnik , I am only back to it! 

The config I had on the connector is basically:
`topics=b.json,es.json`
`topic2TableMap=b.json=b.json,es.json=es.json`

I tried without the `topic2TableMap` initially and got the same result.

**Comment by atillier** [comment]
Right, I figured out the error 27 is from Clickhouse when it's unable to parse a value into a column (I also came across error 60 where it's unable to find a table).

In my case it was because the column `timestamp` doesn't match the format of what I am getting on kafka. 

Hopefully it can help someone else!
